public/js/pimcore/overrides.js

```diff
@@ -400,15 +400,11 @@ Ext.define('pimcore.data.PagingTreeStore', {
                 node.set('expandable', true);
             });
 
-            if (me.pageSize < total || node.inSearch) {
-                node.needsPaging = true;
-                node.pagingData = {
-                    total: data.total,
-                    offset: data.offset,
-                    limit: data.limit
-                }
-            } else {
-                node.needsPaging = false;
```

Comment:
Casually coming from https://github.com/pimcore/admin-ui-classic-bundle/issues/840, is it that `needsPaging` is always true after this PR?